src/gui/log/loglistview.cpp

```diff
@@ -70,13 +70,14 @@ namespace
             painter->save();
             QStyledItemDelegate::paint(painter, option, index); // paints background, focus rect and selection rect
 
-            const QStyle *style = option.widget ? option.widget->style() : QApplication::style();;
+            const QStyle *style = option.widget ? option.widget->style() : QApplication::style();
             const QRect textRect = style->subElementRect(QStyle::SE_ItemViewItemText, &option, option.widget)
                                    .adjusted(1, 0, 0, 0); // shift 1 to avoid text being too close to focus rect
 
-            QFont font = option.font;
-            if (option.font.pointSize() > 0)
-                font.setPointSize(option.font.pointSize()); // somehow this needs to be set directly otherwise painter will use default font
+            QFont font = option.font; // for some reason this doesn't make font a copy of option.font, and we have to manually set font properties
```

Comment:
with scaling 125% and "Make text bigger" 140%

text in yellow is 
> 1 2 3 4 .....  999

without setting font properties manually -
![image](https://user-images.githubusercontent.com/34717789/83292426-b1d1aa00-a207-11ea-9298-74f395e0ee03.png)

before this PR (See the text is not fully visible)
![image](https://user-images.githubusercontent.com/34717789/83292574-f6f5dc00-a207-11ea-9049-d12a6293013f.png)

after this patch (text is fully visible)
![image](https://user-images.githubusercontent.com/34717789/83292709-2efd1f00-a208-11ea-87e0-c5e3b67d1f2e.png)